### PR TITLE
Added brightness adjustment support for Android

### PR DESCRIPTION
**Pull request by Edeneast13**
Implemented brightness adjustment support for Camera2.

### CODE REVIEW

**Comment by n1ru4l** [comment]
@Edeneast13 The build seems to be failing, can you please fix your code?

Also it would be awesome if you could add some documentation about the new brightness feature.

File: src/RNCamera.js [hunk 1]
```diff
@@ -256,6 +262,14 @@ export default class Camera extends React.Component<PropsType, StateType> {
     };
   }
 
+  async setBrightnessAsync(brightness) {
```
Comment (n1ru4l) [inline src/RNCamera.js]: doe we really need these methods if the brightness can already be controlled with the brightness property?

File: src/RNCamera.js [hunk 2]
```diff
@@ -256,6 +262,14 @@ export default class Camera extends React.Component<PropsType, StateType> {
...
+    return await CameraManager.setBrightness(brightness, this._cameraHandle);
+  }
+
+  async getBrightnessAsync() {
```
Comment (n1ru4l) [inline src/RNCamera.js]: same for this method

File: android/.classpath [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (n1ru4l) [inline android/.classpath]: do we need this file in git?

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 4]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```
Comment (n1ru4l) [inline android/.settings/org.eclipse.buildship.core.prefs]: this file seems also not necessary in git

**Comment by n1ru4l** [comment]
@Edeneast13 I added some more comments, could you please resolve the issues and also rebase on master?

**Comment by n1ru4l** [comment]
@Edeneast13 Are you still interested in implementing this? Otherwise I will close it.

**Comment by joshuapinter** [comment]
Us at [CNTRAL](https://github.com/cntral) are interested in implementing this.

**Comment by fabriziobertoglio1987** [comment]
@joshuapinter I am interested in implementing this only for android. I will start working on it soon.

**Pull request closed without merging**